assets/maps/prefabs/prefab_water_honk.dmm

```diff
@@ -75,7 +75,7 @@
 "bw" = (/obj/machinery/vending/monkey,/turf/simulated/floor/carpet,/area/prefab/slimy_honk)
 "bx" = (/turf/simulated/floor/grass{name = "astroturf"},/area/prefab/slimy_honk)
 "by" = (/obj/stool/chair{icon_state = "chair"; dir = 8},/obj/item/reagent_containers/balloon,/turf/simulated/floor/grass{name = "astroturf"},/area/prefab/slimy_honk)
-"bz" = (/obj/table/round/auto,/obj/item/storage/goodybag,/turf/simulated/floor/grass{name = "astroturf"},/area/prefab/slimy_honk)
+"bz" = (/obj/table/round/auto,/obj/item/storage/goodybag,/obj/item/gun/reagent/syringe/love{step_x = 29; step_y = 2},/turf/simulated/floor/grass{name = "astroturf"},/area/prefab/slimy_honk)
```

Comment:
Never use step, always use `pixel_x` and `pixel_y`. This breaks gliding for the entire server

Reply:
Sure!

Reply:
Turns out I'd placed it on the wrong tile anyway, and had just used that to shift it over.

Apologies, never had used the dream maker mapper before, shoulda just sticked with editing the code directly :)